fesod/src/test/java/org/apache/fesod/excel/writesheet/WriteSheetTest.java

```diff
@@ -136,6 +136,13 @@ private void testSheetOrderWithSheetName(ExcelTypeEnum excelTypeEnum) {
             excelWriter.write(dataList(dataMap.get(sheetNo)), writeSheet);
             Assertions.assertEquals(
                     sheetNo, excelWriter.writeContext().writeSheetHolder().getSheetNo());
+
+            sheetNo = 3;
+            writeSheet =
+                    FastExcel.writerSheet(sheetNo, sheetNameList.get(sheetNo)).build();
+            excelWriter.write(dataList(dataMap.get(sheetNo)), writeSheet);
+            Assertions.assertEquals(
+                    sheetNo, excelWriter.writeContext().writeSheetHolder().getSheetNo());
```

Comment:
as this enhancement focus on sheetName substring, how about to assert the name as expectation? sheetNo doesn't show the change point?